src/main/java/net/dries007/tfc/objects/items/ItemStickBundle.java

```diff
@@ -0,0 +1,33 @@
+package net.dries007.tfc.objects.items;
+
+import net.dries007.tfc.api.capability.size.Size;
+import net.dries007.tfc.api.capability.size.Weight;
+import net.dries007.tfc.util.OreDictionaryHelper;
+import net.minecraft.item.ItemStack;
+
+import javax.annotation.Nonnull;
+
+public class ItemStickBundle extends ItemTFC
```

Comment:
This class is redundant. Just use `ItemMisc` and pass in the correct size / ore dictionary names. Mind the required format (e.g. `log_wood`, `stick_bundle`). See other examples for usage.